### PR TITLE
Adds a service image icon for Tidepool Service

### PR DESCRIPTION
**Pull request by rickpasetto**
I had some time over the weekend and started on this work to make the Services menu item a bit nicer in Settings. Like Devices, this adds an icon.

(Note: I did not have access to an "official" Tidepool icon, so I had to screen-snap one from Google Drive)

*Also* This moves the "Secret" share button that kicks off the Acceptance Flow into a more obvious button that indicates it is still under development.

### CODE REVIEW

File: TidepoolServiceKitUI/TidepoolServiceSettingsViewController.swift [hunk 1]
```diff
@@ -123,6 +134,8 @@ final class TidepoolServiceSettingsViewController: UITableViewController, Comple
 
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         switch Section(rawValue: indexPath.section)! {
+        case .startFlow:
+            startFlow()
```
Comment (novalegra) [inline TidepoolServiceKitUI/TidepoolServiceSettingsViewController.swift:138]: [nit] Should probably add a `tableView.deselectRow(at: indexPath, animated: true)` so the cell de-selects correctly

File: TidepoolServiceKitUI/TidepoolServiceSettingsViewController.swift [hunk 2]
```diff
@@ -28,6 +28,8 @@ final class TidepoolServiceSettingsViewController: UITableViewController, Comple
         self.insulinTintColor = insulinTintColor
 
         super.init(style: .grouped)
+        
+        title = "Tidepool Service"
```
Comment (novalegra) [inline TidepoolServiceKitUI/TidepoolServiceSettingsViewController.swift]: [nit] We might want to make this a `NSLocalizedString`

**Review by novalegra — APPROVED** [review]
Two minor nits, otherwise LGTM!

File: TidepoolServiceKitUI/Assets.xcassets/Tidepool Logo.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Tidepool Logo.png",
```
Comment (darinkrauss) [inline TidepoolServiceKitUI/Assets.xcassets/Tidepool Logo.imageset/Contents.json:4]: [nit] This file seems unnecessarily large. Might want to shrink it some.

File: TidepoolServiceKitUI/TidepoolServiceSettingsViewController.swift [hunk 4]
```diff
@@ -96,20 +98,29 @@ final class TidepoolServiceSettingsViewController: UITableViewController, Comple
...
         case .deleteService:
             return " " // Use an empty string for more dramatic spacing
         }
     }
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         switch Section(rawValue: indexPath.section)! {
+        case .startFlow:
+            let cell = tableView.dequeueReusableCell(withIdentifier: TextButtonTableViewCell.className, for: indexPath) as! TextButtonTableViewCell
+            cell.textLabel?.text = LocalizedString("Start Acceptance Flow (under development)", comment: "Button title to start acceptance flow")
```
Comment (darinkrauss) [inline TidepoolServiceKitUI/TidepoolServiceSettingsViewController.swift]: [nit] Doesn't need to be localized since this is only for development.

**Review by darinkrauss — APPROVED** [review]
This seems fine as a temporary workaround. 🥽

**Pull request merged**